### ISSUE TITLE
JPG images merged into PNG, drastically inflating glb size.

### ISSUE THREAD

**Issue by HumanRobot3D** [KhronosGroup/glTF-Blender-IO#1318]
_note: depending on how you look at it this might not be a bug, perhaps the user need more control regarding image format at export for specific image maps._

**Describe the bug**
In specific conditions images might be exported as PNG when they should be exported as JPG, which drastically inflate the size of the resulting GLB/GLTF.

**To Reproduce**/**Expected behavior**
Have one object with one material applied to it.
Feed the BSDF with a Base Color image texture, a bundled Roughness/Metallic map with separated green channel to Roughness and blue channel to Metallic BSDF inputs. (Base Color and R/M are all JPG)
Feed on AO texture to a _glTF Settings_ group node. (AO is JPG)
Set exporter image format to Automatic.
Export.

Two things may happen:
- if your AO texture is of **different** size than your R/M texture they won't get bundled and all of them stay as JPG. That's expected behavior.
- if your AO texture is the **same** size than your R/M texture the AO get bundled on the Red channel and the whole image become PNG. This drastically inflate the overall .glb size for no reason.

In short, it seems that anytime the exporter find AO, R, or M textures of the same size that aren't bundled it will ignore their respective format and merge them into a PNG RGB channels.  This goes against the Automatic format description which state that JPEG will be exported as JPEG and PNG as PNG. 

I don't mind the automatic merging but I would prefer the exporter to check the format of each images before merging and if all are JPG make sure that the resulting image is JPG. To me that's what make the most sense in production.

**Screenshots**
Setup:
![image](https://user-images.githubusercontent.com/28547693/105996670-f4a39480-60aa-11eb-9b7d-4cf461281afd.png)

AO is 256x256, Base Color and R/M are 1024x1024. Result is 3 light JPG (roughly 100ko*3 = 300ko)
![image](https://user-images.githubusercontent.com/28547693/105998077-9081d000-60ac-11eb-9e1e-34fe329bee4b.png)

All textures are 1024x1024. AO get bundled with R/M and resulting image is a heavy PNG (100ko Base Color + 1.14Mo AO/R/M = 1.24Mo)
![image](https://user-images.githubusercontent.com/28547693/105998215-b313e900-60ac-11eb-900c-d0a79a943c41.png)

To be very clear, the size increase is partly due to the AO going from 256 to 1024 but that would be a 100ko difference MAX. The blame is really on the PNG format here.

**.blend file/ .gltf**
[testy-test.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/5880554/testy-test.zip)

**Version**
Windows, last Blender version (2.91.2)

**Additional context**
You might think that I could just force the JPG format at export, but if I had a normal map I'd not want to do that as it need to be PNG.
You might also think that I just have to bundle AO with R/M before exporting to JPG format. And while this would probably work this means that every time I bake a new AO texture I have to go outside of Blender and bundle it manually with R/M. To me that defeat the purpose of the exporter.

**Comment by donmccurdy** [comment]
Does https://github.com/KhronosGroup/glTF-Blender-IO/pull/1099 address this, do you think? There was some discussion of export options and I'm not sure where it ended up.

**Comment by HumanRobot3D** [comment]
Yes this totally address the issue, sorry I couldn't find the post before making mine. And I think you ended up with the right solution avoiding packing JPG with PNG all-together. Any idea when this will be implemented?
